{"name": "test_register[Create user - LastName & username Are Above Max Value]", "status": "failed", "statusDetails": {"message": "AssertionError: FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:41 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1", "trace": "FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:41 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1"}, "attachments": [{"name": "log", "source": "d9fa0cbd-3f31-42b1-b6a9-3d6b5c63ceed-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'Case': 'Create user - LastName & username Are Above Max Value', 'Test Type': 'Negative', 'Request_Body': {'firstName': 'qXKnKjTizXFALNxmACwGYolUmH', 'lastName': 'fXsbXcjEMLmocUdtrJlxMxgKKbXddeVxEyyfwbfKdMheUfjfRox', 'username': 'KVbMcR7149501', 'password': '$[`;}#_)'}}"}], "start": 1670168741401, "stop": 1670168742134, "uuid": "78873fd3-544e-44e1-b618-9188356abf43", "historyId": "9c69021eca76948ae2e2c98c84c4627f", "testCaseId": "bd51a8de3249e024de618e4e17a0a343", "fullName": "tests.test_api.TestApi#test_register", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-EGG762M"}, {"name": "thread", "value": "11972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}